usage-examples/javascript/mongosh/tests/aggregation/group.test.js

```diff
@@ -0,0 +1,256 @@
+const { exec, execSync } = require("child_process");
+const fs = require("fs");
+const path = require("path"); // To help with cross-platform file paths
+const makeTempFileForTesting = require("../../utils/makeTempFileForTesting");
+const unorderedArrayOutputMatches = require("../../utils/unorderedArrayOutputMatches");
+
+jest.setTimeout(10000);
+
+describe("mongosh $group operator tests", () => {
+    const mongoUri = process.env.CONNECTION_STRING;
+    const port = process.env.CONNECTION_PORT;
+    const dbName = "test";
+
+    // Load test data before running the tests
```

Comment:
seems like this will help enforce using standardized sample data across examples too